cognite/client/data_classes/_base.py

```diff
@@ -315,7 +315,7 @@ def dump(self):
 
     @classmethod
     def _get_update_properties(cls):
-        return [key for key in cls.__dict__.keys() if not key.startswith("_")]
+        return [key for key in cls.__dict__.keys() if (not key.startswith("_")) and (not key == "labels")]
```

Comment:
We don't support the `set` operation on assets.labels yet

Reply:
Thanks - this should serve as a workaround for now!
How about this commit string instead: "We currently ignore labels (e.g. leave any labels unchanged) in full updates"
The commit message + the one for the test in   ' test: add test to discard labels on full resource update' were a bit confusing.

Would probably also be good for other users running into this to have a sentence about labels being ignored in full updates in the doc.

Reply:
Great feedback!
I added a new commit to add a sentence to the documentation about the lack of support for labels when performing a full asset replacement. I will also squash-merge this PR and provide a better overall commit message